### PR TITLE
testing/ripgrep: upgrade to 11.0.0

### PR DESCRIPTION
**Pull request by 5paceToast**

### CODE REVIEW

File: testing/ripgrep/APKBUILD [hunk 1]
```diff
@@ -16,17 +16,14 @@ source="$pkgname-$pkgver.tar.gz::https://github.com/BurntSushi/$pkgname/archive/
 builddir="$srcdir/"$pkgname-$pkgver
```
Comment (maxice8) [inline testing/ripgrep/APKBUILD]: This is the default value and thus can be removed

**Comment by Ikke** [comment]
@misery

**Comment by 5paceToast** [comment]
I'd be ok with adopting this, if @misery has no interest in actively keeping it up to date. (I was planning to maintain it anyway, for a while now, as can be seen in my user-aports)

**Comment by misery** [comment]
I'm still interested! But very busy at the moment.

**Comment by misery** [comment]
By the way! 11.0.1 is available.

**Comment by 5paceToast** [comment]
Good catch! Just compiled it on my builder, new PR over at #8373

**Pull request closed without merging**